ironic/drivers/modules/drac/management.py

```diff
@@ -324,27 +324,25 @@ def clear_job_queue(self, task):
         :returns: None if it is completed.
         :raises: RedfishError on an error.
         """
-        system = redfish_utils.get_system(task.node)
-        for manager in system.managers:
-            try:
-                oem_manager = manager.get_oem_extension('Dell')
-            except sushy.exceptions.OEMExtensionNotFoundError as e:
-                error_msg = (_("Search for Sushy OEM extension package "
-                            "'sushy-oem-idrac' failed for node %(node)s. "
-                            "Ensure it is installed. Error: %(error)s") %
-                            {'node': task.node.uuid, 'error': e})
-                LOG.error(error_msg)
-                raise exception.RedfishError(error=error_msg)
-            try:
-                delete_job_response = oem_manager.delete_jobs(task,
-                                                job_ids=['JID_CLEARALL'])
-            except sushy.exceptions.SushyError as e:
-                error_msg = ("Sushy OEM extension Python package "
-                          "sushy-oem-idrac failed to clear job queue"
-                          "for %(node)s, Error : %(error)s" %
-                          {'node':task.node.uuid, 'error':e})
-                LOG.error(error_msg)
-                raise exception.RedfishError(error=error_msg)
+        try:
+            oem_manager = drac_common.get_sushy_oem_manager(task.node)
+        except sushy.exceptions.OEMExtensionNotFoundError as e:
+            error_msg = (_("Search for Sushy OEM extension package "
+			"'sushy-oem-idrac' failed for node %(node)s. "
+		        "Ensure it is installed. Error: %(error)s") %
+		        {'node': task.node.uuid, 'error': e})
+            LOG.error(error_msg)
+            raise exception.RedfishError(error=error_msg)
```

Comment:
Let's move the try/except block into get_sushy_oem_manager().  That way, you can simplify this to just one line here, as well as in the places below where we call get_sushy_oem_manager().

Reply:
Addressed. moved try/except block into get_sushy_oem_manager().